src/transports/dash/low_latency_segment_loader.ts

```diff
@@ -62,7 +62,9 @@ export default function lowLatencySegmentLoader(
     const completeChunks = res[0];
     partialChunk = res[1];
     if (completeChunks !== null) {
-      callbacks.onNewChunk(completeChunks);
+      completeChunks.forEach((completedChunk) => {
+        callbacks.onNewChunk(completedChunk);
+      });
```

Comment:
Is the reburring issue still there? I will have a look if yes

Reply:
No it's fixed :)